src/main/resources/application.conf

```diff
@@ -172,21 +172,16 @@ akka {
     server {
```

Comment:
Removed config that was changed to accommodate the performance issues of "old" slicebox. Hopefully not needed in new versions.